package.json

```diff
@@ -23,6 +23,7 @@
   "bugs": {
     "url": "https://github.com/aquifer/aquifer/issues"
   },
+  "license": "GPL-2.0",
   "licenses": [
```

Comment:
@JeffTomlinson You may want to remove the `licenses` key as well. An older version of the spec allowed for it, but that has since been removed. Have a look at https://docs.npmjs.com/files/package.json#license for more information.